dir.props

```diff
@@ -148,9 +154,6 @@
     <ConfigurationGroup Condition="'$(ConfigurationGroup)' == ''">Debug</ConfigurationGroup>
     <ArchGroup Condition="'$(ArchGroup)' == ''">x64</ArchGroup>
 
-    <!-- For VS support if Configuration is set we default BuildConfiguration based on that property -->
-    <BuildConfiguration Condition="'$(BuildConfiguration)' == '' and '$(Configuration)' != ''">$(Configuration)-$(ArchGroup)</BuildConfiguration>
```

Comment:
That is what will happen now by the defaults. We shouldn't be overridding it with the Configuration property.